docs/components/dropdown/README.md

```diff
@@ -188,6 +188,26 @@ Create a split dropdown button, where the left button provides standard
 <!-- dropdown-split.vue -->
 ```
 
+## Hidden Caret
+The dropdown can be created with the caret hidden by setting the `no-caret` prop to `true`. This is useful when the dropdown is to be displayed as an icon.
+
+**Note:** The caret will always be shown when using `split`
+
+```html
+<div>
+  <b-dropdown variant="link" size="sm" no-caret>
+    <template slot="button-content">
+      <icon icon="avatar"></icon>
```

Comment:
will using `<icon />` here cause issues with the doc generation due to the component not being registered?

Reply:
It won't cause an issue, but it will not show up. 

Since this is a live example (any html section that ends with a comment `<!-- some-name.vue -->` turns into a live example on the docs) it won't render correctly.

I would suggest just using a word, or an HTML entity code for a symbol, in place of the icon component (the docs to not use any icon libraries at the moment)

Reply:
ok thanks. updated to be a unicode emoji icon (magnifying glass)